latest/index.bs

```diff
@@ -2,10 +2,9 @@
 Title: Next-generation file formats (NGFF)
 Shortname: ome-ngff
 Level: 1
-Status: LS-COMMIT
-Status: w3c/ED
+Status: w3c/CG-FINAL
 Group: ome
-URL: https://ngff.openmicroscopy.org/latest/
+URL: https://ngff.openmicroscopy.org/0.2/
```

Comment:
These will need reverting. As a part of https://github.com/ome/ngff/issues/45, this is one of the annoyances of the current static copies.

Reply:
Good point.
I think we should introduce some placeholder for this in the future (https://github.com/ome/ngff/issues/45#issuecomment-826686522), but let's first clean this up manually here.